files-community/Files
# Code Quality: Improved readability of layout page classes

## Summary

As the title.

## Task Checklist

- [x] Improved readability of layout page classes

## Known Issues

_N/A_

## PR Checklist


## Steps To Validate Changes

1. Launch the app
2. See nothing has been changed

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
